tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2017",
+    "target": "ES2019",
```

Comment:
Why is this needed?

Reply:
Don't remember at top of my head but now SB is already using ES2020 in main.

Reply:
reverted